versions.properties

```diff
@@ -621,33 +140,13 @@ version.com.airbnb.android..mvrx=1.5.1
 ##                   # available=2.0.0-beta2
 ##                   # available=2.0.0-beta3
 
-version.com.airbnb.android..mavericks-compose=2.1.0-alpha02
-##                                # available=2.5.1
-##                                # available=2.6.0
-##                                # available=2.6.1
-##                                # available=2.7.0
-##                                # available=3.0.0
-##                                # available=3.0.1
-##                                # available=3.0.2
-##                                # available=3.0.3
+version.com.airbnb.android..mavericks-compose=3.0.3
 
-version.com.airbnb.android..mavericks=2.3.0
-##                        # available=2.4.0
-##                        # available=2.5.0
-##                        # available=2.5.1
-##                        # available=2.6.0
-##                        # available=2.6.1
-##                        # available=2.7.0
-##                        # available=3.0.0
-##                        # available=3.0.1
-##                        # available=3.0.2
-##                        # available=3.0.3
+version.com.airbnb.android..mavericks=3.0.3
```

Comment:
I'm not familiar with the library at all 😬 I didn't know what to expect, but it seems like everything works like expected. I was able to run all sample & repro apps, things seem to be working the same way as they were before the bump

Reply:
I am a maintainer of Mavericks, too 😄 The updates all backwards compatible so we should be good here.